app/models/grda_warehouse/cohort.rb

```diff
@@ -476,12 +487,17 @@ def refresh_time_dependant_client_data(cohort_client_ids: nil)
         end.join('; ')
     end
 
+    private def individual(client)
+      most_recent_enrollment = client.service_history_enrollments.homeless.order(date: :desc).first
+      most_recent_enrollment&.presented_as_individual
+    end
```

Comment:
This should probably be named `individual_in_most_recent_homeless_enrollment` Also, order should be on `first_date_in_program` (which generally will align, but you didn't specify `entry` in the scope, so, you might get an exit date.